source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,270 @@ This changelog summarizes updates to [Mattermost Team Edition](https://mattermos
+ - The advanced logging configuration schema changed. This is a breaking change relative to 5.x. See updated [documentation](https://docs.mattermost.com/comply/audit-log.html).
+ - Some breaking changes to plugins are included:
+   - Support for left-hand side-specific bot icons was dropped.
+   - Removed a deprecated "Backend" field from the plugin manifest.
+   - Converted the "Executables" field in the plugin manifest to a map.
+
+**IMPORTANT:** If you upgrade from a release earlier than v5.39, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+
+### Highlights
+
+#### Multi-Product Support
+ - Mattermost now ships as one platform with three products - Channels, Playbooks, and Boards.
+ - Playbooks and Boards are visible when [plugins are enabled system-wide](https://docs.mattermost.com/configure/configuration-settings.html#enable-plugins). 
+
+#### Global Product Launcher
+ - Added a global header for product navigation for Channels, Playbooks, and Boards. This is disabled on the mobile web view and mobile apps.
+
+#### Branding Changes
+ - Added a new default brand theme named "Denim".
+ - The existing theme names and colors, including "Mattermost", "Organization", "Mattermost Dark", and "Windows Dark" have been updated to the new "Denim", "Sapphire", "Indigo", & "Onyx" theme names and colours, respectively. Anyone using the existing themes will see slightly modified theme colors after their server or workspace is upgraded. The theme variables for the existing "Mattermost", "Organization", "Mattermost Dark", and "Windows Dark" themes will still be accessible in [our documentation](https://docs.mattermost.com/messaging/customizing-theme-colors.html#custom-theme-examples), so a custom theme can be created with these theme variables if desired. Custom themes are unaffected by this change.
+ - Added a new light theme named "Quartz" to the default available list of themes.
+ - Updated email templates to the new branding.
+
+#### Packaging Changes
+ - Updated in-product strings referencing E10 & E20 to [new packaging](https://mattermost.com/pricing).
+ - Features moved from mid SKU to low SKU:
+   - System default permissions, e.g. permission to create and archive channels system-wide.
+      - Specifically, “System Scheme” only in **System Console > User Management > Permissions**. 
+      - Existing permissions/policies in TE/E0 for "Enable Team Creation" and "Allow Team Administrators to edit others’ posts" are properly handled. 
+   - Team and Channel management pages (but without channel moderation, e.g. read-only channels).
+ - Features moved from high SKU to mid SKU:
+   - SSO with OpenID Connect, SAML, Google and O365
+   - O365 integrations including MS Teams Calling and MS Calendar
+   - Jira multi-server support
+   - Advanced team permissions
+   - Read-only announcement channels
+ - Features added to high-end SKU:
```

Comment:
These features are not moving.  I am not certain we need to call them out specifically as they are staying in the Enterprise plan as they were previously.

Reply:
The features listed below to be more concise.